### PR TITLE
Support node cpu orchestration api

### PR DESCRIPTION
**Pull request by eahydra**
### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: apis/scheduling/config/v1beta2/defaults.go [hunk 1]
```diff
@@ -77,9 +77,6 @@ func SetDefaults_NodeNUMAResourceArgs(obj *NodeNUMAResourceArgs) {
 	if obj.DefaultCPUBindPolicy == "" {
 		obj.DefaultCPUBindPolicy = defaultPreferredCPUBindPolicy
 	}
-	if obj.NUMAAllocateStrategy == "" {
-		obj.NUMAAllocateStrategy = defaultNUMAAllocateStrategy
-	}
```
Comment (hormes) [inline apis/scheduling/config/v1beta2/defaults.go:82]: remove the definition?

Comment (eahydra) [reply apis/scheduling/config/v1beta2/defaults.go:82]: done.

**Pull request merged**